source/default_mode/DataNodes.h

```diff
@@ -155,6 +155,8 @@ emp::DataFile & SymWorld::SetUpFreeLivingSymFile(const std::string & filename){
   auto & node7 = GetSymInfectChanceDataNode(); //infect chance
   auto & node8 = GetFreeSymInfectChanceDataNode();
   auto & node9 = GetHostedSymInfectChanceDataNode();
+  auto & node10 = GetInfectionAttemptCount(); //infection rate

```

Comment:
These names should have "DataNode" at the end to be consistent with the others, also let's take the opportunity to improve the names of all of these nodes while we're at it